__tests__/endpoints.test.js

```diff
@@ -41,3 +41,44 @@ describe("Get /api", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id", () => {
+  test(`Responds with:
+
+  an article object, which should have the following properties:
+  author
+  title
+  article_id
+  body
+  topic
+  created_at
+  votes
+  article_img_url`, () => {
+    return request(app)
+      .get("/api/articles/3")
+      .expect(200)
+      .then(({ body }) => {
+        const { article } = body;
+        
+        expect(article).toMatchObject({
+          article_id: expect.any(Number),
+          title: expect.any(String),
+          topic: expect.any(String),
+          author: expect.any(String),
+          body: expect.any(String),
+          created_at: expect.any(String),
+          votes: expect.any(Number),
+          article_img_url: expect.any(String),
+        });
+      });
+  });
+
+  xtest(`should return error if entry doesn't exist`, () => {
+    return request(app)  
+    .get("/api/article/50")
+    .expect(400)
+    .then(({body}) =>{
```

Comment:
I would expect us to do a lot more than just console.log the body. We should also remove any stray console.logs before comitting + pushing & creating a PR